packages/vuetify/src/components/VTreeview/_variables.scss

```diff
@@ -1,4 +1,15 @@
 @use 'sass:map';
 
+$treeview-item-spacer-width: 10px !default;
+$treeview-item-first-icon-margin-start: -6px !default;
+
 $treeview-indent-size: 28px !default; // 28px here to match the width of the expand toggle VBtn.
 $treeview-indent-padding: 16px !default;
+
+$treeview-indent-lines-padding-left: 6px !default;
+$treeview-indent-lines-gap: 0 !default;
+$treeview-indent-lines-opacity: .4 !default;
+$treeview-indent-line-width: 1px !default;
+$treeview-indent-line-color: rgb(var(--v-theme-on-surface)) !default;
+$treeview-indent-line-style: solid !default;
```

Comment:
Explanation for exposing border style and "gap" for vertical padding and a space from `-leaf-link`:

- `double` with 3px width looks surprisingly good
- `dashed` can be combined with `*-lines-gap: 1px` above and looks OK, but depends on the item height